index.html

```diff
@@ -27,13 +27,11 @@
   </section>
   <section data-include="sections/introduction.md" data-include-format="markdown">
   </section>
-  <section data-include="sections/summary.md" data-include-format="markdown">
-  </section>
   <section data-include="sections/jwe.md" data-include-format="markdown">
   </section>
   <section data-include="sections/signing.md" data-include-format="markdown">
   </section>
-  <section data-include="sections/tooling.md" data-include-format="markdown" class="appendix">
+  <section data-include="sections/tooling.md" data-include-format="markdown" class="informative appendix">
```

Comment:
Kunnen we de header van https://github.com/Logius-standaarden/API-mod-encryption/blob/3f0ffcb126d927f945ff4f6b550085bd0763c55b/sections/tooling.md?plain=1#L3 ook weghalen? Die is ook dubbel op